### PR TITLE
ISPN-12950 Grafana dashboard does not differentiate between the cache…

### PR DESCRIPTION
**Pull request by wburns**
…s with same names

https://issues.redhat.com/browse/ISPN-12950

### CODE REVIEW

File: pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go [hunk 1]
```diff
@@ -1409,47 +1409,47 @@ const dashboardJSON = `{
 			},
 			"targets": [
```
Comment (Crumby) [inline pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go:1410]: This panel seems to be duplicate but without repeat configuration

**Comment by Crumby** [comment]
@wburns You missed targets on https://github.com/infinispan/infinispan-operator/blob/6f1679007256f9f056424df2b401070a5a8cfc9c/pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go#L1528

**Comment by wburns** [comment]
Yeah, I am not sure why it is duplicated for some reason. Let me add the extra ones in as well.

**Comment by wburns** [comment]
Updated.

**Review by Crumby — APPROVED** [review]

**Comment by Crumby** [comment]
@wburns I think the replica panels could be deleted and make the json smaller but let's leave it as another issue

File: pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go [hunk 2]
```diff
@@ -1200,47 +1200,47 @@ const dashboardJSON = `{
...
+				"expr": "max({namespace=~\"$namespace\", job=~\"$cluster\", __name__=~\"vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_number_of_entries\"})",
 				"format": "time_series",
 				"instant": false,
 				"intervalFactor": 1,
 				"legendFormat": "Entries",
 				"refId": "A"
 			  },
 			  {
-				"expr": "max({__name__=~\"vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_number_of_entries_in_memory\"})",
+				"expr": "max({namespace=~\"$namespace\", job=~\"$cluster\", __name__=~\"vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_number_of_entries_in_memory\"})",
```
Comment (Crumby) [inline pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go:1211]: Btw this one is now shown in the visualization at all. I think the metric is incorrect (missing `current` word):
Is: `vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_number_of_entries_in_memory`
Should be: `vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_current_number_of_entries_in_memory`

File: pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go [hunk 3]
```diff
@@ -1409,47 +1409,47 @@ const dashboardJSON = `{
...
```
Comment (Crumby) [inline pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go:1420]: Same here

File: pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go (hunk 2 above)
Comment (wburns) [reply pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go:1211]: Ah, yeah the stat name changed at some point :(

File: pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go (hunk 3 above)
Comment (wburns) [reply pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go:1420]: Ah, yeah the stat name changed at some point :(

**Comment by wburns** [comment]
Updated, thanks @Crumby!

**Review by Crumby — APPROVED** [review]

**Comment by rigazilla** [comment]
merged thanks @wburns !

**Pull request merged**